src/blocks/remote-data-container/components/item-list/item-list.tsx

```diff
@@ -32,13 +32,22 @@ export function ItemList( props: ItemListProps ) {
 		return <p>{ __( props.noResultsText ) }</p>;
 	}
 
-	return props.results.map( ( result, index ) => {
-		const blocks =
-			pattern?.blocks.map( block => cloneBlockWithAttributes( block, result, props.blockName ) ) ??
-			[];
-
-		return (
-			<ItemPreview key={ index } blocks={ blocks } onSelect={ () => props.onSelect( result ) } />
-		);
-	} );
+	return (
+		<ul>
```

Comment:
This is kinda unrelated, but I noticed that we're printing `<li>` elements outside of an `<ul>`, `<ol>`, or `<menu>` element which is [out of spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element).

Reply:
Thanks, this was copied from core, but probably missing a container component that supplies the list element.